platform/platform-configuration-listener/src/test/java/org/codice/ddf/configuration/store/SystemConfigurationMigratorTest.java

```diff
@@ -0,0 +1,336 @@
+
+    private static final String KEYSTORE_REL_PATH = "etc/keystores/keystore.jks";
+
+    private static final String KEYSTORE_INVALID_PATH = "/my/ddf/home/../../etc/keystores/keystore.jks";
+
+    private static final String KEYSTORE_ABS_PATH = "/my/ddf/home/etc/keystores/keystore.jks";
+
+    private static final String TRUSTSTORE_REL_PATH = "etc/keystores/truststore.jks";
+
+    private static final String TRUSTSTORE_INVALID_PATH = "/my/ddf/home/../../etc/keystores/truststore.jks";
+
+    private static final String TRUSTSTORE_ABS_PATH = "/my/ddf/home/etc/keystores/truststore.jks";
+
+    private static final String ENC_PROP_FILE_REL_CRL_PATH = "src/test/resources/encryptionCrlRelPath.properties";
+
+    private static final String ENC_PROP_FILE_ABS_CRL_PATH = "src/test/resources/encryptionCrlAbsPath.properties";
+
+    private static final String ENC_PROP_FILE_INVALID_CRL_PATH = "src/test/resources/encryptionCrlInvalidPath.properties";
+
+    private static final String KEYSTORE_SYSTEM_PROP = "javax.net.ssl.keyStore";
+
+    private static final String TRUSTSTORE_SYSTEM_PROP = "javax.net.ssl.trustStore";
+
+    private static final String WS_SECURITY_DIR_REL_PATH = "etc/ws-security";
+
+    private static final String PDP_POLICIES_DIR_REL_PATH = "etc/pdp";
+
+    private static final String SYSTEM_PROPERTIES_REL_PATH = "etc/system.properties";
+
+    private static final String USERS_PROPERTIES_REL_PATH = "etc/users.properties";
+
+    private static final String VALID_CRL_REL_PATH_1 = "etc/certs/demoCA/crl/crl.pem";
+
+    @Before
+    public void setup() {
+        mockStatic(System.class);
+        mockStatic(FileUtils.class);
+    }
+
+    @Test
+    public void testExportValidRelativePaths() throws Exception {
+        // Setup
+        when(System.getProperty(KEYSTORE_SYSTEM_PROP)).thenReturn(KEYSTORE_REL_PATH);
+        when(System.getProperty(TRUSTSTORE_SYSTEM_PROP)).thenReturn(TRUSTSTORE_REL_PATH);
+        Stream<String> stream = getStream(ENC_PROP_FILE_REL_CRL_PATH);
+        doNothing().when(FileUtils.class);
+        SystemConfigurationMigrator securityConfigurationMigrator = new SystemConfigurationMigrator(
+                DDF_HOME) {
+            @Override
+            Stream<String> getStream(Path path) throws IOException {
+                return stream;
+            }
+        };
+
+        // Perform Test
+        securityConfigurationMigrator.export(EXPORT_DIRECTORY);
+
+        // Verify
+        ArgumentCaptor<File> sourceDirCaptor = ArgumentCaptor.forClass(File.class);
+        ArgumentCaptor<File> destinationDirCaptor = ArgumentCaptor.forClass(File.class);
+        verifyStatic(times(2));
+        FileUtils.copyDirectory(sourceDirCaptor.capture(), destinationDirCaptor.capture());
+        List<File> sourceDirs = sourceDirCaptor.getAllValues();
+        List<File> destinationDirs = destinationDirCaptor.getAllValues();
+        assertThat(sourceDirs.get(0).getAbsolutePath(), is(DDF_HOME + File.separator
```

Comment:
You may be able to do `assertThat(sourceDirs, containsInAnyOrder(...))` here instead of having separate assertions.